som/gensom.cpp

```diff
@@ -3,16 +3,15 @@
 
```

Comment:
This file should not be part of your commit. It changed upstream. Please remove this file from the commit.